CMakeLists.txt

```diff
@@ -149,9 +168,13 @@ add_subdirectory(third_party/OpenGL-Registry EXCLUDE_FROM_ALL)
 # TODO(paulthomson): Probably remove this and use a different loading strategy/library.
 add_subdirectory(third_party/glad EXCLUDE_FROM_ALL)
 
+if(NOT SHADERTRAP_SKIP_LODEPNG)
 add_subdirectory(third_party/lodepng EXCLUDE_FROM_ALL)
+endif()
```

Comment:
I guess the stuff inside an if ... endif should be indented (a few instances of this in the PR)

Reply:
done